src/main/java/hu/oe/nik/szfmv/automatedcar/model/IWorld.java

```diff
@@ -0,0 +1,24 @@
+package hu.oe.nik.szfmv.automatedcar.model;
```

Comment:
hu.oe.nik.szfmv.automatedcar.model.interface-be kérlek

Reply:
az interface nem valid név,
```java
hu.oe.nik.szfmv.automatedcar.model.interfaces
```
jó lesz?

Reply:
igen, bocsi